packages/lib-classifier/src/components/Classifier/components/Layout/helpers/getLayout/getLayout.js

```diff
@@ -4,7 +4,6 @@ export default function getLayout(layout) {
   if (layouts[layout]) {
     return layouts[layout]
   } else {
-    console.warn(`Couldn't find a layout for '${layout}', falling back to default`)
```

Comment:
Opinionated statement, but this warning was just cluttering the console during unit tests.